crates/wit-component/src/decoding.rs

```diff
@@ -871,7 +871,7 @@ impl WitPackageDecoder<'_> {
                     .context("failed to convert anonymous result type")?,
             )
         } else {
-            Results::Named(
+            Results::Named(Params(
```

Comment:
Could implement `Into` and `From` trait so don't need to wrap a `Params` outside but can call `into()`.